lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -613,8 +613,8 @@ def verify_flow_rules_deletion(self):
             dpath, {"table_id": self.SPGW_TABLE},
         )
         assert(
-            len(flows) == 2
-        ), "There should only be 2 default table 0 flows"
+            len(flows) == 4
+        ), "There should only be 4 default table 0 flows"
```

Comment:
Is it needed for all test cases? Have we recently introduced 2 new default flow rules which would be there even after UE is detached?

Reply:
This was after effect of PR #11101